authorize_helper.go

```diff
@@ -175,3 +182,118 @@ func IsLocalhost(redirectURI *url.URL) bool {
 	hn := redirectURI.Hostname()
 	return strings.HasSuffix(hn, ".localhost") || hn == "127.0.0.1" || hn == "localhost"
 }
+
+func WriteAuthorizeFormPostResponse(redirectURL string, parameters url.Values, rw io.Writer) {
+	t := template.Must(template.New("form_post").Parse(`<html>
```

Comment:
This template could be defined as a global variable which means that it would not need to be rendered for every request!

Reply:
+1